modules/ROOT/pages/deploying-to-cloudhub.adoc

```diff
@@ -158,278 +160,287 @@ Deploying myMuleApp ...
+image::cloudhub-deploy-app-sandbox.png[Application Name field and Get from sandbox and Deploy Application buttons]
+. Provide a name for your application and click *Get from sandbox*:
++
+This button is available only if a non-production environment exists.
+. Select your sandbox environment and select the application to move to production.
++
+Only applications that are deployed to CloudHub appear in the list.
+You can promote only applications deployed to CloudHub to run on other CloudHub workers.
+. If you want to copy the environment variables and Mule version from the source application, select *Merge environment variables and mule version*.
+. Click *Apply*.
+. Click tabs to configure application options.
++
+See <<configure-application-settings,Configure Application Settings>>.
+. Click *Deploy Application*.
++
+See <<Deployment Progress>>.
 
-Worker sizes have different compute and memory capacities, including:
+== Application Names 
 
-[%header,cols="2*a"]
-|===
-|Worker Size |Heap Memory
-|0.1 vCores |500 MB
-|0.2 vCores |1 GB
-|1 vCore |1.5 GB
-|2 vCores |3.5 GB
-|4 vCores |7.5 GB
-|8 vCores |15 GB
-|16 vCores |32 GB
-|===
+The application name identifies your application in Runtime Manager and also used for the 
+public application name in the `cloudhub.io` domain.
+For example, an application named `myapplication` is accessible at `+http://myapplication.cloudhub.io+`.
 
-Workers that have less than 1 vCore capacity (0.1 vCores and 0.2 vCores) offer limited CPU and IO for smaller work loads. Each worker has 8 GB of storage, which is used for both system and application storage.
+The application name must be globally unique across CloudHub to avoid domain conflicts.
+Runtime Manager prevents you from choosing a name that is already in use.
 
-[NOTE]
---
-Workers running on fractional vCores (0.1 vCores and 0.2 vCores) have the ability to burst to higher CPU speeds for a short duration of time. This is useful for improving application start-up times and with processing infrequently large workloads. Allocating workers with 1 vCore or greater is recommended if consistency of performance is needed.
---
+To avoid domain conflicts, create a naming convention for your applications, such as adding a prefix to 
+your applications: `mycompany-mydept-myapplication`.
+You can then add DNS records to hide this complex application name.
+For example, you can route requests to `myapplication.mycompany.com` to `mycompany-mydept-myapplication.cloudhub.io`.
 
-Applications with greater storage needs (for example, verbose logging) should use one of the larger worker sizes--2 vCores or 4 vCores, which have additional storage as follows:
+With the CloudHub dedicated load balancer, you can optionally implement mapping rules to set your application's public URL to any other path, as long as you own the domain.
```

Comment:
Is this restating what the DNS records do?

Reply:
...and is this part of avoiding duplicate names?

Reply:
No and no.
The DLB does the mapping between internal and external.